### PR TITLE
Topology Id Rework Design

### PR DESCRIPTION
**Pull request by wburns**

### CODE REVIEW

File: TopologyId.asciidoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+This means a client will have a _higher_ topology so the server will never send the cluster information
+and the client will, until the server toplogy catches up, no longer know what servers a key map to and
+also send to servers that may not even exist any longer.
+
+=== Suggested Fix
+
+Fixing this will require a few different things.
+
+* Change the toopology id to be a more unique number that can be recreated instead of an incrementing number
+** Possible choices are to hash the server member ids
```
Comment (pruivo) [inline TopologyId.asciidoc:22]: better hashing the mappings (segments => owners) since the state transfer uses 2 topology changes.

Comment (wburns) [reply TopologyId.asciidoc:22]: Yes, sorry I didn't mean this as the answer but just a suggestion. I should have tried to go into more specifics. Let me update with some actual options.

**Comment by pruivo** [comment]
@wburns is it ok to merge this?

**Comment by wburns** [comment]
Sure

**Pull request merged**